_posts/2021-07-30-release-1.12.5.md

```diff
@@ -0,0 +1,210 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23223'>FLINK-23223</a>] -         When flushAlways is enabled the subpartition may lose notification of data availability
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23233'>FLINK-23233</a>] -         OperatorEventSendingCheckpointITCase.testOperatorEventLostWithReaderFailure fails on azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23248'>FLINK-23248</a>] -         SinkWriter is not closed when failing
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23266'>FLINK-23266</a>] -         HA per-job cluster (rocks, non-incremental) hangs on Azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23362'>FLINK-23362</a>] -         ClientTest.testSimpleRequests fails due to timeout on azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23417'>FLINK-23417</a>] -         MiniClusterITCase.testHandleBatchJobsWhenNotEnoughSlot fails on Azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23429'>FLINK-23429</a>] -         State Processor API failed with FileNotFoundException when working with state files on Cloud Storage
+</li>
+</ul>
+
+<h2>        Improvement
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17857'>FLINK-17857</a>] -         Kubernetes and docker e2e tests could not run on Mac OS after migration
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18182'>FLINK-18182</a>] -         Upgrade AWS SDK in flink-connector-kinesis to include new region af-south-1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20695'>FLINK-20695</a>] -         Zookeeper node under leader and leaderlatch is not deleted after job finished
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21229'>FLINK-21229</a>] -         Support ssl connection with schema registry format
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21411'>FLINK-21411</a>] -         The components on which Flink depends may contain vulnerabilities. If yes, fix them.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22708'>FLINK-22708</a>] -         Propagate savepoint settings from StreamExecutionEnvironment to StreamGraph
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22747'>FLINK-22747</a>] -         Update commons-io to 2.8
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22757'>FLINK-22757</a>] -         Update GCS documentation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22774'>FLINK-22774</a>] -         Update Kinesis SQL connector&#39;s Guava to 27.0-jre
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22799'>FLINK-22799</a>] -         Missing documentation: State backwards compatibility is not support for Flink SQL
```

Comment:
This issue has been marked as duplicated and should be removed from release note.